payment/manual/renew-manual.go

```diff
@@ -84,6 +83,21 @@ func RenewManualPaymentFx(w http.ResponseWriter, r *http.Request) (string, inter
 		return "", nil, err
 	}
 
+	networkNode = network.GetNetworkNodeByUid(policy.ProducerUid)
+	if networkNode == nil {
+		err = errors.New("networkNode not found")
+		return "", nil, err
+	}
+	warrant := networkNode.GetWarrant()
+	flowName, _ := policy.GetFlow(networkNode, warrant)
+	log.Printf("flowName '%s'", flowName)
+
+	isOnlineProducer := authToken.UserID != policy.ProducerUid || flowName != models.RemittanceMgaFlow
+	if authToken.IsNetworkNode && isOnlineProducer {
+		err = errors.New("cannot access to transaction")
+		return "", nil, err
+	}
```

Comment:
Just to remind you that if you change the logic in the other handler to update it here too.